### ISSUE TITLE
[Reanimated] Cannot import findHostInstance_DEPRECATED.

### ISSUE THREAD

**Issue by gitmazzio** [software-mansion/react-native-reanimated#5455]
### Description

I updated react-native to 0.72.7 and many libraries including this one to 3.6.1 and and I activated Fabric I've been getting this error it gives me an error.

`[Reanimated] Cannot import findHostInstance_DEPRECATED.`

Could you help me?

To be clear, I also had this error with version 3.5.4 of react-native-reanimated but I only tried yesterday to update to the new version to see if it had already been resolved

![Screenshot_1701441816](https://github.com/software-mansion/react-native-reanimated/assets/29253427/74781359-99a6-4265-972b-3afe50dd35c9)

### Steps to reproduce

I have Animated.View or Animated.ScrollView.

### Snack or a link to a repository

none

### Reanimated version

3.6.1

### React Native version

0.72.7

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native

### Architecture

Fabric (New Architecture)

### Build type

Debug mode

### Device

None

### Acknowledgements

Yes

**Comment by gitmazzio** [comment]
> @gitmazzio I'm afraid this is some kind of config error on your side, we test the library on Fabric and it works for us. Can you confirm that you see `"fabric":true` in your Metro bundler console when you launch the app?

I didn't notice that there was no `fabric: true`, I also looked at your FabricExample project and I am unable to activate it. 
Do you have advice? I'm desperate and maybe I'm missing something.

**Comment by tomekzaw** [comment]
@gitmazzio How did you "activate" Fabric? Is your app compliant with official RN 0.72.7 template?

**Comment by gitmazzio** [comment]
@tomekzaw on IOS fabric is enabled correctly. In Android project I put `newArchEnabled=true` into `gradle.properties` file.

**Comment by sn4f** [comment]
Hi, in my case this one helped me:

metro.config.js

```
const config = {};
```

replaced with

```
const config = {
  transformer: {
    getTransformOptions: async () => ({
      transform: {
        inlineRequires: true,
      },
    }),
  },
};
```

**Comment by sumitmanral** [comment]
I am getting the same error in RN 0.72

**Comment by piaskowyk** [comment]
This issue should be fixed with this PR - https://github.com/software-mansion/react-native-reanimated/pull/5588 It will be available in reanimated@3.7.0
